Header header

string[] name
float64[] estimatedAcceleration
float64[] desiredAcceleration
float64[] commandedAcceleration
float64[] errorAcceleration
float64[] sumErrorAcceleration
